lib/commands/new.js

```diff
@@ -64,6 +71,13 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError(message));
     }
 
+    let langDef = this.availableOptions.find((opt) => opt.name === 'lang').default;
+    let langInfo = getLangArgInfo(commandOptions.lang);
+    if (langInfo.langArgValue !== langDef && !langInfo.result) {
+      return Promise.reject(new SilentError(langInfo.message));
```

Comment:
Since we are already in an async function, there is no need to `return Promise.reject` here. Doing this is the same thing:

```suggestion
      throw new SilentError(langInfo.message);
```

Reply:
This got removed when consolidating lang-related logic into init in f95d3a9, but I did make the corresponding change int init. I haven't yet updated the utility throw the error internally.